### ISSUE TITLE
Migrate to Vite

### ISSUE THREAD

**Issue by Dharmik48** [Dun-sin/anon-chat-app#54]
The initial build and load times in `create-react-app` are very long. The app takes a long time to start up.

On the other hand, using `vite`, the times are reduced by a significant amount! So, I think migrating to `vite` can really enhance developer's experience.

**Comment by Dun-sin** [comment]
Yeah that's true I use vite for my other project and it's very fast.

Do you want to work on it?

**Comment by Dharmik48** [comment]
Yeah, sure! I'd love to!
